<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpinBoxDemo</class>
 <widget class="QMainWindow" name="SpinBoxDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpinBox测试程序</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>数值输入输出</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>数量</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="numberSpin">
           <property name="suffix">
            <string> kg</string>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>单价</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QDoubleSpinBox" name="perSpin">
           <property name="prefix">
            <string>$ </string>
           </property>
           <property name="value">
            <double>9.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>总价</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QDoubleSpinBox" name="totalSpin">
           <property name="prefix">
            <string>$ </string>
           </property>
           <property name="maximum">
            <double>999.990000000000009</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="calBtn">
           <property name="text">
            <string>计算</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>进制转换</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>二进制</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>八进制</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>十进制</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>十六进制</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QSpinBox" name="binSpin">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string/>
             </property>
             <property name="prefix">
              <string>Bin </string>
             </property>
             <property name="minimum">
              <number>-10000</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="displayIntegerBase">
              <number>2</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="octSpin">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string/>
             </property>
             <property name="prefix">
              <string>Oct </string>
             </property>
             <property name="minimum">
              <number>-10000</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="displayIntegerBase">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="decSpin">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string/>
             </property>
             <property name="prefix">
              <string>Dec </string>
             </property>
             <property name="minimum">
              <number>-10000</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="hexSpin">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string/>
             </property>
             <property name="prefix">
              <string>Hex </string>
             </property>
             <property name="minimum">
              <number>-10000</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="displayIntegerBase">
              <number>16</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
